list.md

```diff
@@ -85,6 +85,9 @@ Deutsche Zentralbibliothek für Medizin (ZB MED)
 UB Ilmenau
 [https://github.com/ubilmenau](https://github.com/ubilmenau)
 
+project finc
```

Comment:
@hbunke and all other contributors: maybe a new heading for projects with an own GitHub account could be added to `list.md`, something like `Bibliothekarische Projekte`. Then projects like `finc` could be added there. Ideally such projects are not restricted to one institution, but IMHO it is fine to name the institution where a project was started or is mainly based. `https://github.com/goobi/` is another example which would fit under the new heading.